requirements.txt

```diff
@@ -1,5 +0,0 @@
-# Library so -e .
```

Comment:
why dont we need this anymore?

Reply:
For libraries our recommended configuration is to put dependencies in setup.py and not have a requirements.txt:

> Libraries should not have deps = -rrequirements.txt in their tox files. Having a requirements.txt is not recommended in libraries.

https://yelpwiki.yelpcorp.com/display/WEBCORE/Applications+vs+Libraries